plugins/org/src/components/Cards/Group/MembersList/MembersListCard.tsx

```diff
@@ -147,20 +154,35 @@ export const MembersListCard = (_props: {
   return (
     <Grid item>
       <InfoCard
-        title={`Members (${members?.length || 0})`}
+        title={`Members (${members?.length || 0}) ${
+          members && members.length > pageSize
```

Comment:
I wonder if we can extract this stuff out into variables inside the component to make the JSX a little cleaner.

And then we can re-use things like `Math.ceil((members?.length || 0) / pageSize)` instead of duplicating it down at the bottom in the `count` prop passed to the `Pagination` component.

Reply:
OK, I created 2 const: 1 for the nb of pages (Math.ceil...)  and one with the pagination Label

The label is displayed if needed. It look like: 

![screenshot-localhost_7000-2021 05 07-19_23_21](https://user-images.githubusercontent.com/24318548/117486447-f2828900-af69-11eb-95e2-d83ae9334a38.png)